spec/models/order_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Order, type: :model do
+  before(:each) do
+    @client = FactoryGirl.create(:client)
```

Comment:
@DmitrS I think, it's better to write one line `client` in orders' factory :wink: